helix-core/src/test/java/org/apache/helix/model/TestClusterConfig.java

```diff
@@ -298,6 +298,26 @@ public void testAsyncGlobalRebalanceOption() {
             false), true);
   }
 
+  @Test
+  public void testGetMaintenanceOfflineNodeTimeOut() {
```

Comment:
1. We need a test to cover the get default value case.
2. Might be easier for you to combine the 2 new tests.

Reply:
Sounds good.